.github/renovate.json5

```diff
@@ -0,0 +1,51 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended",
+        ":approveMajorUpdates",
+        ":semanticCommitScopeDisabled"
+    ],
+    "internalChecksFilter": "strict",
+    "labels": ["dependencies"],
+
+    // Wait a few days for any new package, as a precaution against malicious publishes
+    "minimumReleaseAge": "7 days",
+
+    "packageRules": [
+        {
+            "description": "Automerge minor version updates. Major versions must be approved from the Dependency Dashboard issue, because we use the :approveMajorUpdates preset.",
+            "automerge": true,
```

Comment:
Does automerge work only for in-range updates? In particular, we sometimes need to pin dependencies (for example, https://github.com/eslint/eslint/pull/16897) and in that case further updates shouldn't be automatically merged.

Reply:
This setting will work for all passing updates - in-range or otherwise. You can perhaps skip automerging initially until more familiar with Renovate's approach?

Reply:
Not currently - Renovate doesn't expose metadata about in-range vs not

Reply:
Okay, then in such exceptional cases that rarely happen, we'll probably need to temporarily configure Renovate to ignore the problematic package, which seems fine to me.

Reply:
> Okay, then in such exceptional cases that rarely happen, we'll probably need to temporarily configure Renovate to ignore the problematic package, which seems fine to me.

If you want to "ignore" a dependency, I recommend setting it to `dependencyDashboardApproval=true`. This means you can always get the update from Renovate's Dependency Dashboard. For example:

```json
{
  "packageRules": [
    {
      "matchPackagePatterns": ["^@somescope"],
      "dependencyDashboardApproval": true
    }
  ]
}
```

Some users ignore a dependency by disabling the package manager in Renovate. That means you don't see anything about the update. With my method above, you'll always be reminded about the update on the Dependency Dashboard.